controllers/secrets.go

```diff
@@ -45,7 +45,14 @@ func (r *TrustyAIServiceReconciler) findDatabaseSecret(ctx context.Context, inst
 // validateDatabaseSecret validates the DB configuration secret
 func (r *TrustyAIServiceReconciler) validateDatabaseSecret(secret *corev1.Secret) error {
 
-	mandatoryKeys := []string{"databaseKind", "databaseUsername", "databasePassword", "databaseService", "databasePort"}
+	mandatoryKeys := []string{
+		"databaseKind",
+		"databaseUsername",
+		"databasePassword",
+		"databaseService",
+		"databasePort",
+		"databaseName",
```

Comment:
Do we want to have a default fallback to `trustyai_database` to be backward compatible?

Reply:
Yeah, I agree